### DISCUSSION TITLE
I dont know how to install it

### DISCUSSION THREAD

**Discussion by MirdjusCZ** [Daksh777/SpotifyNoPremium#116] [Q&A]
I am not very into these thinks, but if I install the .bat file and I run it, it starts to write something in the command line and then what? It writes press any key to continue, but thats all. What I have to do to actually install it? I thought it will install the spotify but it didnt. 
Thanks for response.

**Comment by Daksh777** [comment]
Hi @MirdjusCZ, really sorry for the late reply.
I've been quite busy with work and personal stuff.

Do you still need help?
